src/quo2/components/settings/accounts/view.cljs

```diff
@@ -0,0 +1,46 @@
+(ns quo2.components.settings.accounts.view
+  (:require [quo.react-native :as rn]
+            [quo2.components.avatars.account-avatar :as av]
```

Comment:
better to use fill name alias `[quo2.components.avatars.account-avatar :as account-avatar ]`

Reply:
Totally agree!
Solved in 3d9a6bf0afe3430696defb1e18fa8de2d9a94eaa